prov/efa/src/efa.h

```diff
@@ -418,6 +418,10 @@ int efa_cq_open(struct fid_domain *domain_fid, struct fi_cq_attr *attr,
 		struct fid_cq **cq_fid, void *context);
 int efa_fabric(struct fi_fabric_attr *attr, struct fid_fabric **fabric_fid,
 	       void *context);
+int efa_getinfo(uint32_t version, const char *node, const char *service,
+		uint64_t flags, const struct fi_info *hints, struct fi_info **info);
+
+void fi_efa_fini(void);
```

Comment:
This function name need to change to something like `efa_prov_fini` to conform to the convention the all function name start with `efa`.

Reply:
Made that change, and updated. Thanks!